mindmeld/active_learning/classifiers.py

```diff
@@ -168,9 +172,70 @@ def _validate_class_level_statistic(class_level_statistic):
             )
         return class_level_statistic
 
+    @staticmethod
+    def _get_entity_probs(
+        classifier: Classifier,
+        queries: ProcessedQueryList,
+        tag_to_id: Dict,
+    ):
+        """Get the probability distribution for a query across domains or intents.
+
+        Args:
+            classifier (MindMeld Classifer): Domain or Intent Classifier
+            queries (ProcessedQueryList): List of MindMeld queries
+            tag_to_id (Dict): Dictionary mapping domain or intent names to vector index
+                positions.
+
+        Returns:
+            prob_vector (List[List]]): Probability distribution vectors for given queries.
+        """
+        queries_prob_vectors = []
+        if queries:
+            classifier_eval = classifier.evaluate(queries=queries, active_learning=True)
+
+            domain = classifier.domain
+            intent = classifier.intent
+            default_prob = 1.0
+            default_tag = "O|"
+            default_key = f"{domain}.{intent}.{default_tag}"
+            default_idx = tag_to_id[f"{default_key}"]
+
+            if not classifier_eval:
+                for _ in range(len(queries)):
+                    query_prob_vector_2d = np.zeros((1, len(tag_to_id)))
+
+                    query_prob_vector_2d[0][default_idx] = default_prob
+                    queries_prob_vectors.append(query_prob_vector_2d)
+
+            else:
```

Comment:
To reduce indentation, we can remove the "else" condition here and add "return queries_prob_vectors" within the if block